### PR TITLE
[T4A3][W09-B1] Ng Yong Sheng

### PR DESCRIPTION
**Pull request by JasonScourge**

### CODE REVIEW

**Comment by JasonScourge** [comment]
Kept the equals methods but overrode it in the various classes because it does not make sense if an Address value with the exact same String is the same as a Phone value, which will happen if they use the same equals method.

**Comment by JasonScourge** [comment]
Kept the equals methods but overrode it in the various classes because it does not make sense if an Address value with the exact same String as a Phone value be equated the same with the parent class equals method, which will happen if they use the same equals method.

**Comment by JasonScourge** [comment]
Remove the comments from the Contact class, the 2nd, 3rd line.
Change the first line of comment to something relevant.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -4,28 +4,25 @@
 
 /**
  * Represents a Person's address in the address book.
+ * Inherits from the Contact Class
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java:7]: The line can be omitted, as that info is already captured clearly in line 10.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's address in the address book.
+ * Serves as a parent class for other classes
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Contact.java:7]: This line can be omitted. Any class can serve as a parent class for other classes (unless it is declared as `final`)

**Comment by damithc** [comment]
Good effort. 
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**